x-pack/plugins/ingest_manager/public/plugin.ts

```diff
@@ -95,6 +99,21 @@ export class IngestManagerPlugin
       deps.home.tutorials.registerDirectoryNotice(PLUGIN_ID, TutorialDirectoryNotice);
       deps.home.tutorials.registerDirectoryHeaderLink(PLUGIN_ID, TutorialDirectoryHeaderLink);
       deps.home.tutorials.registerModuleNotice(PLUGIN_ID, TutorialModuleNotice);
+
+      deps.home.featureCatalogue.register({
+        id: 'ingestManager',
+        title: i18n.translate('xpack.ingestManager.featureCatalogueTitle', {
+          defaultMessage: 'Add Elastic Agent',
+        }),
+        description: i18n.translate('xpack.ingestManager.featureCatalogueDescription', {
+          defaultMessage: 'Add and manage your fleet of Elastic Agents and integrations.',
+        }),
+        icon: 'logstashInput',
```

Comment:
@hbharding @mostlyjason Is the icon we want to use? We've been using `savedObjectsApp` inside Ingest Manager, should we use the same here?

Reply:
I think we decided as the gear icon as a temporary icon https://github.com/elastic/Design/issues/2469

Reply:
@mostlyjason: I believe we're currently using a gear icon for the link to stack management in the top right of the page currently. Would it make sense to use an alternative icon to represent Ingest Manager to differentiate itself from stack management as a whole?